website/docs/reference/global-configs/behavior-changes.md

```diff
@@ -164,3 +166,12 @@ In previous versions (dbt Core 1.8 and earlier), the MetricFlow time spine confi
 When the flag is set to `True`, dbt will continue to support the SQL file configuration. When the flag is set to `False`, dbt will raise a deprecation warning if it detects a MetricFlow time spine configured in a SQL file. 
 
 The MetricFlow YAML file should have the `time_spine:` field. Refer to [MetricFlow timespine](/docs/build/metricflow-time-spine) for more details. 
+
+### Custom microbatch strategy
+The `require_batched_execution_for_custom_microbatch_strategy` flag is set to `False` by default and is only relevant if you already have a custom microbatch macro in your project.  If you don't have a custom microbatch macro, you don't need to set this flag as dbt will handle microbatching automatically once you set the [microbatch strategy](/docs/build/incremental-microbatch) in your project.
```

Comment:
--TODO LINK TO OTHER COMMENT--
Same change request as in `website/docs/docs/build/incremental-microbatch.md`